src/Core/NotificationHub/NotificationHubPool.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Bit.Core.Settings;
+using Bit.Core.Utilities;
+using Microsoft.Azure.NotificationHubs;
+using Microsoft.Extensions.Logging;
+
+namespace Bit.Core.NotificationHub;
+
+public class NotificationHubPool : INotificationHubPool
+{
+    private List<NotificationHubConnection> _connections { get; }
+    private readonly IEnumerable<INotificationHubClient> _clients;
+    private readonly ILogger<NotificationHubPool> _logger;
+    public NotificationHubPool(ILogger<NotificationHubPool> logger, GlobalSettings globalSettings)
+    {
+        _logger = logger;
+        _connections = filterInvalidHubs(globalSettings.NotificationHubPool.NotificationHubSettings);
+        _clients = _connections.Select(c => c.HubClient);
+    }
+
+    private List<NotificationHubConnection> filterInvalidHubs(IEnumerable<GlobalSettings.NotificationHubSettings> hubs)
```

Comment:
Why camel case method name?

Reply:
Because I'm too used to typescript, thank you 😳